### PR TITLE
more or less working gRPC

### PR DESCRIPTION
**Pull request by dominikvrbic**
@frle10 can you check wtf went wrong the scheduler is not getting the message even tho I am sending it let me know if you need more info I have yet to document any of this and writer tests

### CODE REVIEW

File: scheduler/scheduler.go [hunk 1]
```diff
@@ -1,15 +1,31 @@
...
 func main() {
 	http.HandleFunc("/hello", helloHandler)
 	http.ListenAndServe(":8081", nil)
+
 }
 
 func helloHandler(w http.ResponseWriter, r *http.Request) {
-	fmt.Fprint(w, "Hello, World!")
+	var conn *grpc.ClientConn
+	conn, err := grpc.Dial(":8080", grpc.WithTransportCredentials(insecure.NewCredentials()))
```
Comment (frle10) [inline scheduler/scheduler.go:21]: Externalize port maybe?

Comment (frle10) [inline scheduler/scheduler.go:21]: Externalize port maybe?

File: go.work.sum
Comment (frle10) [inline go.work.sum:1]: What is the purpose of this file? I've never seen a go.work.sum, only go.sum which we already have.

Comment (dominikvrbic) [reply go.work.sum:1]: we are using Golang workspaces so it is a workspace-level sum file, so when you do an install they can share the deps

Comment (dominikvrbic) [reply go.work.sum:1]: same as a top lvl yarn lock in a monorepo

File: scheduler/scheduler.go (hunk 1 above)
Comment (dominikvrbic) [reply scheduler/scheduler.go:21]: i would do that when we add make files and env setup

Comment (dominikvrbic) [reply scheduler/scheduler.go:21]: since it should not be port but url anyhow

Comment (frle10) [reply scheduler/scheduler.go:21]: Sounds good

**Review by frle10 — APPROVED** [review]

**Pull request merged**